x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/MlInitializationService.java

```diff
@@ -112,5 +142,113 @@ MlDailyMaintenanceService getDailyMaintenanceService() {
         return mlDailyMaintenanceService;
     }
 
+    private void makeMlIndicesHidden() {
+        String[] mlHiddenIndexPatterns = MachineLearning.getMlHiddenIndexPatterns();
+
+        CountDownLatch latch = new CountDownLatch(1);
+
+        LatchedActionListener<Boolean> listener = new LatchedActionListener<>(
+            ActionListener.wrap(
+                result -> logger.info("All the ML internal indices and aliases were successfully made hidden"),
+                e -> logger.error("An error occurred while making ML internal indices and aliases hidden", e)
+            ),
+            latch
+        );
+
+        ActionListener<AcknowledgedResponse> updateAliasesListener = ActionListener.wrap(
+            updateAliasesResponse -> {
+                if (updateAliasesResponse.isAcknowledged() == false) {
+                    logger.error("One or more of the following ML internal aliases could not be made hidden.");
```

Comment:
The message says "of the following" but no list of index names follows.  So either "of the following" needs to be removed from this message or else a list needs to be appended.